geopandas/array.py

```diff
@@ -84,6 +84,31 @@ def _check_crs(left, right, allow_none=False):
     return True
 
 
+def _crs_mismatch_warn(left, right, stacklevel=1):
```

Comment:
```suggestion
def _crs_mismatch_warn(left, right, stacklevel=3):
```

The default maybe doesn't matter much if it is not used, but in practice it will always be at a minimum 3